docs/example/README.md

```diff
@@ -214,6 +227,48 @@ Wait for the changes to propagate and observe the counters increment for booksto
 
 - http://localhost:8082 - **bookstore-v2**
 
+## Permissive Traffic Policy Mode
+
+In permissive mode, application connectivity within the mesh is automatically configured by `osm-controller`. It can be enabled in the following ways.
+
+1. During install using `osm` cli:
+```bash
+osm install --enable-permissive-traffic-policy
+```
+
+2. Post install by updating the `osm-config` ConfigMap in the control plane's namespace (`osm-system` by default)
+```bash
+# Replace osm-system with osm-controller's namespace if using a non default namespace
+kubectl patch ConfigMap osm-config -n osm-system -p '{"data":{"permissive_traffic_policy_mode":"true"}}' --type=merge
+```
+
+### Deploy a service with permissive mode enabled
+
+Enable permissive mode as described above and verify that the counters on the `Bookbuyer` window are not incrementing anymore.
+
+Then deploy a new `bookstore` service which has not been deployed previously. The `bookstore` deployment has been configured with an identity of `bookstore-v1` for the purpose of this demo:
```

Comment:
Why is this a separate deployment from docs/example/manifests/apps/? ie. can we update the one in apps with this one?

Reply:
This is separate because the manifest for bookbuyer deployment has a different label. Keeping it separate to avoid messing the existing manifests.

Reply:
This extra step to deploy a separate yaml is not ideal. Can you expand on why merging these would create issues? In the real world, my application workload would have a pod and a service and the label is only applied when I'm ready to join the namespace to the mesh. ideally the demo should reflect real-world step by step.

Reply:
Discussed offline, doc has been updated.